packages/react-scripts/package.json

```diff
@@ -18,7 +18,8 @@
     "scripts",
     "template",
     "template-typescript",
-    "utils"
+    "utils",
+    "backpack-addons"
```

Comment:
Re-install success in banana:
![image](https://user-images.githubusercontent.com/86700834/156289513-7d4772f6-2901-4b54-9fee-ed4482cd1aec.png)